rcore/protocol/protocol_blob.c

```diff
@@ -40,7 +40,7 @@ typedef struct {
 void printblock(void *data, size_t len)
 {
     for(int i = 0; i < len; i++)
-        printf("%02x", *(((uint8_t *)data) + i));
```

Comment:
This one is trickier, since it's formatting a whole block as things that are not individual log messages.  Does anyone use `printblock`?

Reply:
ack'ed the whole project, only found printblock in the same file.